iron-menu-behavior.html

```diff
@@ -208,24 +208,50 @@
     },
 
     /**
+     * Returns the item in the first menu from where the event was dispatched.
+     *
+     * @param {Event}
+     */
+    _isClosestNode: function(event) {
+      var path = Polymer.dom(event).path.slice();
+      var items = this.items;
+      var node;
+
+      while ((node = path.shift()) && !node._isClosestNode) {
+        var i = items.indexOf(node);
+        if (i >= 0) {
+          return {
+            value: this._indexToValue(i),
+            item: node
+          };
+        }
+      }
+      return null;
+    },
+
+    /**
      * Handler that is called when a shift+tab keypress is detected by the menu.
      *
      * @param {CustomEvent} event A key combination event.
      */
     _onShiftTabDown: function(event) {
-      var oldTabIndex;
+      if (this._isClosestNode(event.detail.keyboardEvent)) {
```

Comment:
Rather than indenting the whole method implemetation, just `return` early if the condition is not `true`.